blosc2/blosc2_ext.pyx

```diff
@@ -941,6 +942,8 @@ cdef class SChunk:
         if kwargs is not None:
             if self.mode == "w":
                 blosc2.remove_urlpath(urlpath)
+            elif self.mode == "r" and (urlpath is None or not os.path.exists(urlpath)):
+                raise ValueError("SChunk must already exist")
```

Comment:
I have discussed this with Marta, and we have had a hard time understanding this message.  What about "When `mode` is 'r', SChunk must already exist on-disk; if not, you cannot specify `mode`".
Would that be informative enough?

Reply:
True, this message was not super clear^^
I expanded the message and also made it explicit whether no path was given or the file does not exist.